alert_source.go

```diff
@@ -0,0 +1,57 @@
+package opslevel
+
+import (
+	"github.com/shurcooL/graphql"
+)
+
+type AlertSourceExternalIdentifier struct {
+	Type 			 AlertSourceTypeEnum	`graphql:"type" json:"type"`
```

Comment:
These technically only need the `json` struct tag as they are used for input only.  More context can be found here - https://github.com/OpsLevel/opslevel-go/pull/75#discussion_r939462528